packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -332,30 +338,21 @@ class DropdownCore extends React.Component<Props, State> {
     // Figure out focus states for the dropdown after it has changed from open
     // to closed or vice versa
     initialFocusItem() {
-        const {
-            keyboard,
-            initialFocusedIndex,
-            open,
-            onSearchTextChanged,
-            searchText,
-        } = this.props;
+        const {open, onSearchTextChanged, searchText} = this.props;
 
         if (open) {
-            // Reset focused index
-            this.focusedIndex = initialFocusedIndex;
```

Comment:
So, I just broke this behavior and I don't know how to fix it - the ability to specify which item in the list to focus initially is in conflict with the behavior of focusing the menu container when it is opened.

It might be possible to focus the focusedIndex if one is specified in props? And default to focusing the menu? And warn about breaking a11y best practices by setting it in the documentation?

Reply:
(related - I commented out a bunch of tests above that were at least partly about this behavior that obviously need addressing once we know what the plan is)

Reply:
Oh, just a heads up... @PeppeMiller is working on fixing the focus when the dropdown is opened, so there might be some things you both would like to align on: https://khanacademy.atlassian.net/browse/WB-1091